Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN mkdir -p /usr/src/protoc/ \
 # Install GRPC and Protobuf.
 RUN pip3 install --upgrade pip \
   && pip3 install \
-    grpcio==1.3.0 \
+    grpcio==1.3.5 \
```

Comment:
It is not the end of the world if it does not; artman will uninstall the old one and install the new one. It makes the image slightly smaller if they match. :-)